api/dp-openapi.yaml

```
@@ -1370,6 +1547,74 @@ components:
       items:
         type: string
 
+    proof_page:
+      type: object
+      properties:
+        text:
+          description: The page text. If page state is 'out' from OCR or previous round, else from current round
+          type: string
+        pagestate:
+          description: The page state, for checking continuity. Either out or temp
+          type: string
+        saved:
+          description: True if the page state is 'temp'
+          type: boolean
+        pagename:
+          description: The filename of the page with extension
+          type: string
+        image_url:
+          description: URL of the image of the page
+          type: string
+        language_direction:
+          description: left-to-right or right-to-left
+          type: string
+          enum: [RTL, LTR]
+        pagenum:
+          description: Page "number" without extension
+          type: string
+        round_info_array:
```

To match what you're returning:
```suggestion
        round_info:
```